py-polars/polars/lazyframe/frame.py

```diff
@@ -2128,10 +2131,8 @@ def sink_csv(
         >>> lf.sink_csv("out.csv")  # doctest: +SKIP
 
         """
-        if len(separator) != 1:
-            raise ValueError("only single byte separator is allowed")
-        if len(quote) != 1:
-            raise ValueError("only single byte quote char is allowed")
+        _check_arg_is_1byte("separator", separator, can_be_empty=False)
+        _check_arg_is_1byte("quote_char", quote_char, can_be_empty=False)
```

Comment:
Nice, I'm always happy to see existing utils be applied to reduce duplicated logic 👌